src/encoded/tests/data/workbook-inserts/experiment_hi_c.json

```diff
@@ -72,6 +72,10 @@
         "lab": "test-4dn-lab",
         "aliases": ["labsample:alias3"],
         "files": ["4DNFIO67APZ1", "4DNFIO67AZZ1"],
+        "other_processed_files": [{
+            "title": "Test FileFastq as opf",
+            "files": ["/files-fastq/4DNFIO67AZB1/"]
+            }],
```

Comment:
Codacy found an issue: [Expected indentation of 8 spaces but found 12.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6115226)